### ISSUE TITLE
ssh ProxyCommand

### ISSUE THREAD

**Issue by technicalguy** [MisterTea/EternalTerminal#155]
Can I use EternalTerminal with ssh's ProxyCommand to get

    localhost -> A -> B

where A is only running ssh and B is running ssh+et?

**Comment by ailzhang** [comment]
No, A has to run ssh+et as well to get the connection back automatically.
